apps/prairielearn/elements/pl-external-grader-results/pl-external-grader-results.py

```diff
@@ -141,16 +147,16 @@ def render(element_html, data):
                             )
```

Comment:
Same comment as for the `html_params`. A bunch of the stuff loaded into the `test` dictionary can be done on the initial construction, instead of all on separate lines.